### DISCUSSION TITLE
Help integrating google map.

### DISCUSSION THREAD

**Discussion by mituBarua** [Budibase/budibase#4355] [Help]
Anyone help me how can i integrate google map in budibase? or any resources?

**Comment by mjashanks** [comment]
It depends on what you need to do with it, but it's probably not possible right now. Your best bet is to use the `Embed` component, with a `<script>` tag inside.

The issue is that you can feed data **into** the embed component, but you can't get data back out. I.e. you won't be able to get the "Current Location" of the map.
